doc/odoc_for_authors.mld

```diff
@@ -663,6 +663,34 @@ v}
 In this case the reference text would be "Foo Bar" so the paragraph would read
 "See Foo Bar for details".
 
+Sections with a label cannot be referenced via the section title, only
```

Comment:
Currently, they can but that's a new behavior from 2.0 and might be removed or might be kept. I think we can remove this sentence until we made a decision on this.

Reply:
Do you mean it is possible on `master` right now?

I've tested this odoc 2.0.0 and they cannot be referenced or at least not with the usual syntax. My impression was that the manual should be up to date with the version of the source it lives in, i.e. `master` here.

From a user perspective I think this should be documented since I assumed otherwise by default and was a bit surprised when I tried it and saw it fail.

I can remove it if you prefer but I'm not sure the fact that it might change in the future is a good reason not to document the current behaviour and syntax.

Reply:
Just to be clear: are you expecting to be able to reference the following section:
```
{1:label Heading}
```
by `{!label}` _and_ by `{!heading}`?

Reply:
I was initially. Now that I know I can't it makes total sense. I'm not implying that it should behave any differently but just that the existing documentation or lack of, made assume it would accept both, hence, I think, the need for clarification.